src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1012,9 +1014,11 @@ protected File prepareTempDockerfile(File dockerfile) throws PluginExecutionExce
             debug("Creating temp Dockerfile...");
             tempDockerfile = File.createTempFile("tempDockerfile", "");
             debug("temp Dockerfile: " + tempDockerfile);
-            tempDockerfilePath = tempDockerfile.toPath();
-            // set the tempDockerfile to be deleted when the JVM exits
-            tempDockerfile.deleteOnExit();
+            tempDockerfilePath = tempDockerfile.toPath(); // save name to clean up later
```

Comment:
If `keepTempDockerfile` is enabled, it would also be useful to print out an `info` message on what the temp Dockerfile path is.

Reply:
Ok, that is easier than finding the name in the docker build command.

Reply:
[INFO] Saving temporary Dockerfile named:/var/folders/p1/v7svjg7s6xngg377xtsjxcmc0000gn/T/tempDockerfile7677461533838714336